frontend/simple/js/state.js

```diff
@@ -74,8 +74,7 @@ const mutations = {
   },
   pending (state, contractId) {
     if (!state.contracts[contractId]) {
-      let index = state.pending.indexOf(contractId)
-      index === -1 && state.pending.push(contractId)
+      !state.pending.includes(contractId) && state.pending.push(contractId)
```

Comment:
@taoeffect I made a change that still uses the `if` statement because I feel in this case it is more intuitive, let me know what you think